R/RcppExports.R

```diff
@@ -331,14 +415,26 @@ swe_azalt_rev <- function(jd_ut, coord_flag, geopos, xin) {
     .Call(`_swephR_azalt_rev`, jd_ut, coord_flag, geopos, xin)
 }
 
+#' @details
+#' \describe{
```

Comment:
The warning comes from this line. Indeed, the brace opened on this line is never closed. However, this is a generated file ...